### PR TITLE
Update pipeline release v0.56.x

### PR DESCRIPTION
**Pull request by chitrangpatel**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Thanks @chitrangpatel, can you also remove v0.54.x since it's now EOL?

**Comment by chitrangpatel** [comment]
> Thanks @chitrangpatel, can you also remove v0.54.x since it's now EOL?

Done! Thanks

**Review by AlanGreene — APPROVED** [review]
/lgtm

File: sync/config/pipelines.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ archive: https://github.com/tektoncd/pipeline/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
-- name: release-v0.54.x
-  displayName: v0.54.x
+- name: release-v0.56.x
```
Comment (AlanGreene) [inline sync/config/pipelines.yaml:29]: This branch doesn't exist in tektoncd/pipeline 👀

**Review by AlanGreene — CHANGES_REQUESTED** [review]

File: sync/config/pipelines.yaml (hunk 1 above)
Comment (chitrangpatel) [reply sync/config/pipelines.yaml:29]: Ah Thanks for the catch. I had named it `...v0.56.0` I just renamed it to `...v0.56.x`

Comment (chitrangpatel) [reply sync/config/pipelines.yaml:29]: Is that why the tests below were failing?

Comment (AlanGreene) [reply sync/config/pipelines.yaml:29]: Yes that's the reason

**Comment by AlanGreene** [comment]
@chitrangpatel I've rerun the netlify jobs and they're passing: https://deploy-preview-585--tekton.netlify.app/docs/
However it's not updating the GitHub checks, can you update the PR to re-trigger them please?

**Review by AlanGreene — APPROVED** [review]
Thanks @chitrangpatel 
/lgtm

**Pull request merged**